golang/x/relay/client/cli/tx.go

```diff
@@ -68,6 +69,46 @@ func GetCmdIngestHeaderChain(cdc *codec.Codec) *cobra.Command {
 	}
 }
 
+// GetCmdIngestDifficultyChange creates a CLI command to ingest a difficulty change
+func GetCmdIngestDifficultyChange(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "ingestdiffchange <prev epoch start> <json list of headers>",
+		Short: "Ingest a difficulty change",
+		Long:  "Ingest a difficulty change",
```

Comment:
Should specify that `prev epoch start` is in hex, and whether it's raw or a digest